mpt/impl/imap-mailbox/core/src/main/resources/org/apache/james/imap/scripts/Annotation.test

```diff
@@ -141,4 +141,19 @@ C: g18 GETMETADATA (/private/comment)
 S: g18 BAD GETMETADATA failed. Illegal arguments.
 
 C: g19 GETMETADATA "INBOX" (MAXSIZE 0) (/private/comment)
-S: g19 BAD GETMETADATA failed. Illegal arguments.
\ No newline at end of file
+S: g19 BAD GETMETADATA failed. Illegal arguments.
+
+########################End of get annotation############
+
+C: a21 SETMETADATA INBOX (/private "My new comment")
+S: a21 BAD SETMETADATA failed. Illegal arguments.
+
+C: a22 SETMETADATA INBOX (/private/vendor "My new comment")
+S: a22 BAD SETMETADATA failed. Illegal arguments.
```

Comment:
In fact, it had on another ticket when doing comparison with cyrus. I agree with you as end user perspective